CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contribute to JaRIKo
+
+Feedback and pull requests are very welcome!
+
+## Ask for help, feature or report a bug
+
+Feel free to create feature request and bug report in the [issues](https://github.com/smeup/jariko/issues).
```

Comment:
It could make sense to add a way to send us questions:
* a gitter channel
* a slack channel
* an email address

Reply:
Gitter an slack are good if they have some volume of traffic.
For the email address, I have to do some checks :-)